cms/templates/js/signatory-details.underscore

```diff
@@ -0,0 +1,22 @@
+<div class="signatory-panel-default">
+  <div class="actions certificate-actions signatory-panel-edit">
+    <span class="action action-edit-signatory">
+      <a href="javascript:void(0);" class="edit-signatory"><i class="icon fa fa-pencil" aria-hidden="true"></i> <%= gettext("Edit") %></a>
+    </span>
+  </div>
+  <div class="signatory-panel-header">Signatory <%= signatory_number %></div>
+  <div class="signatory-panel-body">
+    <div>
+      <span class="signatory-name-label"><%= gettext("Name: ") %></span>
```

Comment:
I would suggest a translator note that it is important to keep the space. It would be better though if you could do this as a single string with a placeholder to make translation more robust. Not sure if that is possible with the classes.